glade/XML.custom

```diff
@@ -25,15 +25,20 @@
 // Boston, MA 02111-1307, USA.
 
 		// keep this around so it doesn't get GC'd
-		static GladeSharp.XMLCustomWidgetHandlerWrapper callback_wrapper = null;
+		static GCHandle gch;
+		static Glade.XMLCustomWidgetHandler callback_wrapper = null;
 
 		[DllImport("libglade-2.0-0.dll", CallingConvention=CallingConvention.Cdecl)]
 		static extern void glade_set_custom_handler (GladeSharp.XMLCustomWidgetHandlerNative handler, IntPtr user_data);
 
 		public static Glade.XMLCustomWidgetHandler CustomHandler {
 			set {
-				callback_wrapper = new GladeSharp.XMLCustomWidgetHandlerWrapper (value);
-				glade_set_custom_handler(callback_wrapper.NativeDelegate, IntPtr.Zero);
+				if (callback_wrapper != null) {
+					gch.Free ();
+				}
+
+				gch = GCHandle.Alloc (value);
+				glade_set_custom_handler(GladeSharp.XMLCustomWidgetHandlerWrapper.NativeDelegate, (IntPtr)gch);
```

Comment:
Don't we need to set `callback_wrapper`? It'll always be null here, right? Should this be changed to:
```
- static Glade.XMLCustomWidgetHandler callback_wrapper = null;
...
...
if (gch.IsAllocated)
    gch.Free ();
```

Reply:
gch.IsAllocated only works after free, afaik

Reply:
Fixed.